docs/getstarted/userinterface.md

```diff
@@ -279,6 +279,38 @@ You can Drag and Drop editor groups on the workbench, move individual Tabs betwe
 
 >**Note:** VS Code uses editor groups whether or not you have enabled Tabs.  Without Tabs, editor groups are a stack of your open items with the most recently selected item visible in the editor pane.
 
+### Locked editor groups
+
+By default, files will always open in the active editor group. Locked editor groups help to avoid this. The `workbench.editor.autoLockGroups` setting allows you to select editor types that should lock a group automatically when they open. This only applies when the editor is the first to open in an otherwise empty or new group.
+
+```json
+"workbench.editor.autoLockGroups": {
+  "default": false,
+  "terminalEditor": true
+}
+```
+
+Terminals are configured by default to cause a new group to lock automatically. A lock icon in the action toolbar (top right) indicates a locked group. The locked state of an editor group is persisted and restored across restarts.
+
+<!-- Required? Locked editor https://code.visualstudio.com/assets/updates/1_61/editor-readonly-deleted.png -->
+
+You can control the lock state of editor groups with these commands:
+
+- `workbench.action.lockEditorGroup` - locks a group
+- `workbench.action.unlockEditorGroup` - unlocks a group
+- `workbench.action.toggleEditorGroupLock` - toggles the current lock state of the group
+
+Locked groups behave differently than unlocked groups:
+
+- New editors will not open in a locked group unless explicitly moved there.
+- If an editor skips a locked group, the new editor will either open in the most recently used unlocked group or create a new group to the side of the locked one.
+
+If you have more than one editor group opened, you can lock it manually from the "..." overflow menu or using the **View: Toggle Editor Group Lock** or **View: Lock Editor Lock** commands. 
+
+You can also lock empty groups for a more stable editor layout. With setting `workbench.editor.closeEmptyGroups` you can control whether an empty group is closed when the last tab in a group is closed. When disabled, empty groups will remain part of the grid.  
+
+<!-- Required?  Lock group option in overflow menu https://code.visualstudio.com/assets/updates/1_60/locked-editor-group.png -->
```

Comment:
still open. If needed, can someone move the images to the correct location and create the image link?